master/main.go

```diff
@@ -165,25 +246,62 @@ func sendGraphToWorker(graph graphs.Graph, worker *worker, parameters map[string
 		Headers: map[string]string{"Content-Type": "application/json"},
 		Params:  paramsMapToRequestParamsMap(parameters),
 	}
-	_, err := grequests.Post(worker.Address+"/graph", &options)
+	resp, err := grequests.Post(worker.Address+"/graph", &options)
+	defer resp.Close()
```

Comment:
Ik denk dit na de niet nill check gedaan moet worden, volgensmij is resp namelijk nil als er een error is, dan resulteert dit natuurlijk in een nilpointer exception.